pkgs/applications/editors/neovim/default.nix

```diff
@@ -79,9 +79,8 @@ in
       "-DGPERF_PRG=${gperf}/bin/gperf"
       "-DLUA_PRG=${neovimLuaEnv.interpreter}"
     ]
-    # FIXME: this is verry messy and strange.
-    ++ optional (!stdenv.isDarwin) "-DLIBLUV_LIBRARY=${lua.pkgs.luv}/lib/lua/${lua.luaversion}/luv.so"
-    ++ optional (stdenv.isDarwin) "-DLIBLUV_LIBRARY=${lua.pkgs.luv.libluv}/lib/lua/${lua.luaversion}/libluv.dylib"
+    # TODO: Check if that's needed on darwin
```

Comment:
This isn't needed, and fails the build with:
```
make[2]: *** No rule to make target '/nix/store/gml3nvcfnsrb73n12kwndh3rcad5kpfg-luv-1.34.1-1/lib/lua/5.1/libluv.dylib', needed by 'bin/nvim'.  Stop.
```

The build passes without this.

Reply:
Fixed in `6a1604285f87da05ea716ecbac14f1ad3f47f6ed`.